lib/widgets/summary_widget/main/main_text_count.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/cupertino.dart';
+
+class TextCountWidget extends StatelessWidget {
+  final _count;
+  final double _fontSize;
+  TextCountWidget(this._count, this._fontSize);
```

Comment:
I recommend using { } and @required in every widgets that has 2 or more parameters.

Reply:
Noted